u2f-gae-demo/war/js/u2f-api.js

```diff
@@ -223,6 +225,14 @@ u2f.isAndroidChrome_ = function() {
 };
 
 /**
+ * Detect chrome running on iOS based on the browser's platform.
+ * @private
+ */
+u2f.isIOSChrome_ = function() {
```

Comment:
Nit: Common usage of camel case with acronyms would tend to prefer Ios over IOS, here and throughout.